plugins/quickbooks-online/src/src/Facade/QuickBooksFacade.php

```diff
@@ -283,14 +283,31 @@ public function exportInvoices(): void
                     } else {
                         $TaxCode = 'NON';
                     }
-
+                    
+                    
+                    $referencedIdOfLine = $qbItem->Id;
```

Comment:
Going to have to do some testing to see what the ucrm api returns
but looks good! Thanks 

Also need to check the code because it looks like it adds a new product on every line item anyway, so will have to sync products and services separately and only add new products if the line item isn't one of the above.